app/src/pages/inside/projectSettingsPageContainer/content/notifications/modals/addEditNotificationModal/addEditNotificationModal.jsx

```diff
@@ -241,32 +238,45 @@ const AddEditNotificationModal = ({
     },
   ];
 
-  const {
-    actionType,
-    notification: { sendCase, informOwner },
-  } = data;
+  const { actionType } = data;
 
-  const okButton = {
-    text: formatMessage(COMMON_LOCALE_KEYS.SAVE),
-    onClick: () => {
-      const modalName = `${messages.title.defaultMessage.replace(
-        /\{.*\}/i,
-        messages[actionType].defaultMessage,
-      )}`;
+  const submitActions = (formFieldValues) => {
+    const modalName = `${messages.title.defaultMessage.replace(
+      /\{.*\}/i,
+      messages[actionType].defaultMessage,
+    )}`;
+
+    const newFormValues = { ...formFieldValues };
+
```

Comment:
No needed spam with empty line if you define variables or functions. use it if you want split some peace of code. highlight something significant(usually if\else statements, return in the function, etc)
not a call to refactor this code